Failing to install the repo with mamba on Windows 11 OS

When installing the repo I get the following error
```
LINK : fatal error LNK1181: cannot open input file 'C:\Program.obj' [C:\Users\apaolino\code\ironcub_mk1_software\build\lib\gazebo\plugins\jets-plugin\jets-plugin-v1.vcxproj]
```

Output of `mamba list` command

```
# packages in environment at C:\Users\apaolino\AppData\Local\mambaforge\envs\robsub:
#
# Name                    Version                   Build  Channel
ace                       7.1.1                h63175ca_0    conda-forge
aom                       3.6.1                h63175ca_0    conda-forge
assimp                    5.3.1                h81f0834_2    conda-forge
blockfactory              0.8.3              h63175ca_101    robotology
blosc                     1.21.5               hdccc3a2_0    conda-forge
bullet-cpp                3.25                 hf63dbb6_2    conda-forge
bzip2                     1.0.8                hcfcfb64_5    conda-forge
c-ares                    1.21.0               hcfcfb64_0    conda-forge
c-compiler                1.6.0                hcfcfb64_0    conda-forge
ca-certificates           2023.7.22            h56e8100_0    conda-forge
cairo                     1.18.0               h1fef639_0    conda-forge
catkin_pkg                1.0.0              pyhd8ed1ab_0    conda-forge
cfitsio                   4.3.0                h9b0cee5_0    conda-forge
clangdev                  5.0.0                   flang_3    conda-forge
cmake                     3.27.6               hf0feee3_0    conda-forge
compilers                 1.6.0                h57928b3_0    conda-forge
console_bridge            1.0.2                h5362a0b_1    conda-forge
cppzmq                    4.10.0               h42135b4_1    conda-forge
cxx-compiler              1.6.0                h91493d7_0    conda-forge
dartsim                   6.13.0               haebf736_5    conda-forge
dav1d                     1.2.1                hcfcfb64_0    conda-forge
dirent                    1.21                          0    conda-forge
dlfcn-win32               1.3.0                h0e60522_0    conda-forge
docutils                  0.20.1          py311h1ea47a8_2    conda-forge
eigen                     3.4.0                h91493d7_0    conda-forge
expat                     2.5.0                h63175ca_1    conda-forge
fcl                       0.7.0                he22821c_4    conda-forge
ffmpeg                    6.0.0           gpl_h1627b0f_105    conda-forge
flang                     5.0.0           he025d50_20180525    conda-forge
flang_win-64              5.0.0           h13ae965_20180526    conda-forge
flann                     1.9.2                h23e6bae_0    conda-forge
fmt                       10.1.1               h181d51b_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.2               hbde0cde_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fortran-compiler          1.6.0                h9655429_0    conda-forge
freeglut                  3.2.2                h63175ca_2    conda-forge
freeimage                 3.18.0              h609497f_18    conda-forge
freetype                  2.12.1               hdaf720e_2    conda-forge
freexl                    2.0.0                h8276f4a_0    conda-forge
fribidi                   1.0.10               h8d14728_0    conda-forge
gazebo                    11.14.0              h7b4f576_0    conda-forge
gazebo-yarp-plugins       4.9.0                h57928b3_0    conda-forge
geos                      3.12.0               h1537add_0    conda-forge
geotiff                   1.7.1               hcf4a93f_14    conda-forge
getopt-win32              0.1                  hcfcfb64_1    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
git                       2.42.0               h57928b3_0    conda-forge
git-lfs                   3.4.0                h57928b3_0    conda-forge
glfw                      3.3.8                hcfcfb64_2    conda-forge
glib                      2.78.1               h12be248_0    conda-forge
glib-tools                2.78.1               h12be248_0    conda-forge
graphite2                 1.3.13                     1000    conda-forge
graphviz                  8.1.0                h51cb2cd_0    conda-forge
gsl                       2.7                  hdfb1a43_0    conda-forge
gst-plugins-base          1.22.6               h001b923_2    conda-forge
gstreamer                 1.22.6               hb4038d2_2    conda-forge
gts                       0.7.6                h6b5321d_4    conda-forge
harfbuzz                  8.2.1                h7ab893a_0    conda-forge
hdf4                      4.2.15               h5557f11_7    conda-forge
hdf5                      1.14.2          nompi_h73e8ff5_100    conda-forge
icu                       73.2                 h63175ca_0    conda-forge
icub-firmware-shared      1.36.0             h63175ca_101    robotology
icub-main                 2.3.0           py311hf8ab716_101    robotology
icub-models               2.4.1           py311h12c1d0e_0    conda-forge
idyntree                  10.0.0          py311h0a7ed4b_0    conda-forge
idyntree-matlab-bindings  10.0.0             h63175ca_101    robotology
imath                     3.1.9                h12be248_0    conda-forge
intel-openmp              2023.2.0         h57928b3_50497    conda-forge
ipopt                     3.14.12              ha9547d1_1    conda-forge
irrlicht                  1.8.5                h65f4d7e_4    conda-forge
jasper                    4.1.0                h28f2b1a_0    conda-forge
jsoncpp                   1.9.5                h2d74725_1    conda-forge
jxrlib                    1.1                  h8ffe710_2    conda-forge
kealib                    1.5.2                ha10e780_1    conda-forge
khronos-opencl-icd-loader 2023.04.17           h64bf75a_0    conda-forge
krb5                      1.21.2               heb0366b_0    conda-forge
lcms2                     2.15                 h67d730c_3    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libabseil                 20230802.1      cxx17_h63175ca_0    conda-forge
libaec                    1.1.2                h63175ca_1    conda-forge
libarchive                3.7.2                h6f8411a_0    conda-forge
libblas                   3.9.0              19_win64_mkl    conda-forge
libboost                  1.82.0               h65993cd_6    conda-forge
libboost-devel            1.82.0               h91493d7_6    conda-forge
libboost-headers          1.82.0               h57928b3_6    conda-forge
libcblas                  3.9.0              19_win64_mkl    conda-forge
libccd-double             2.1                  h63175ca_3    conda-forge
libclang                  15.0.7          default_h77d9078_3    conda-forge
libclang13                15.0.7          default_h77d9078_3    conda-forge
libcrc32c                 1.1.2                h0e60522_0    conda-forge
libcurl                   8.4.0                hd5e4a3a_0    conda-forge
libdeflate                1.19                 hcfcfb64_0    conda-forge
libexpat                  2.5.0                h63175ca_1    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libflang                  5.0.0           h6538335_20180525    conda-forge
libgazebo-yarp-plugins    4.9.0                h545c7f3_0    conda-forge
libgd                     2.3.3                h312136b_9    conda-forge
libgdal                   3.7.3                h3217549_2    conda-forge
libglib                   2.78.1               he8f3873_0    conda-forge
libgoogle-cloud           2.12.0               ha74b051_3    conda-forge
libgrpc                   1.58.2               h2a9c87f_0    conda-forge
libhwloc                  2.9.3           default_haede6df_1009    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libignition-cmake2        2.16.0               h63175ca_1    conda-forge
libignition-common3       3.15.1               hc1f3b3c_2    conda-forge
libignition-fuel-tools4   4.6.0                hadaf4a5_4    conda-forge
libignition-math6         6.15.0          py311h4b297e5_1    conda-forge
libignition-msgs5         5.11.0               h90b4331_3    conda-forge
libignition-tools1        1.5.0                hf5993cb_3    conda-forge
libignition-transport8    8.4.0                h4ff85d8_6    conda-forge
libjpeg-turbo             3.0.0                hcfcfb64_1    conda-forge
libkml                    1.3.0             haf3e7a6_1018    conda-forge
liblapack                 3.9.0              19_win64_mkl    conda-forge
liblapacke                3.9.0              19_win64_mkl    conda-forge
libnetcdf                 4.9.2           nompi_h8284064_112    conda-forge
libode                    0.16.2              h12c1d0e_12    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libopenblas               0.3.24          pthreads_hc140b1d_0    conda-forge
libopencv                 4.8.1           py311h6be31c5_4    conda-forge
libopenvino               2023.1.0             h63175ca_1    conda-forge
libopenvino-auto-batch-plugin 2023.1.0             h63175ca_1    conda-forge
libopenvino-auto-plugin   2023.1.0             h63175ca_1    conda-forge
libopenvino-hetero-plugin 2023.1.0             h63175ca_1    conda-forge
libopenvino-intel-cpu-plugin 2023.1.0             h63175ca_1    conda-forge
libopenvino-intel-gpu-plugin 2023.1.0             h63175ca_1    conda-forge
libopenvino-ir-frontend   2023.1.0             h63175ca_1    conda-forge
libopenvino-onnx-frontend 2023.1.0             h63175ca_1    conda-forge
libopenvino-paddle-frontend 2023.1.0             h63175ca_1    conda-forge
libopenvino-pytorch-frontend 2023.1.0             h63175ca_1    conda-forge
libopenvino-tensorflow-frontend 2023.1.0             h63175ca_1    conda-forge
libopenvino-tensorflow-lite-frontend 2023.1.0             h63175ca_1    conda-forge
libopus                   1.3.1                h8ffe710_1    conda-forge
libosqp                   0.6.3                h63175ca_0    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libpq                     16.0                 h43585b0_1    conda-forge
libprotobuf               4.24.3               hb8276f3_1    conda-forge
libqdldl                  0.1.5                h63175ca_1    conda-forge
libraw                    0.21.1               h5557f11_2    conda-forge
libre2-11                 2023.06.02           h8c5ae5e_0    conda-forge
librttopo                 1.1.0               h92c5fdb_14    conda-forge
libsdformat               9.8.0                h5ba8b87_5    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libspatialite             5.1.0                hbf340bc_1    conda-forge
libsqlite                 3.44.0               hcfcfb64_0    conda-forge
libssh2                   1.11.0               h7dfc565_0    conda-forge
libtiff                   4.6.0                h6e2ebb7_2    conda-forge
libusb                    1.0.26             h8ffe710_100    conda-forge
libuv                     1.44.2               hcfcfb64_1    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp                   1.3.2                hcfcfb64_1    conda-forge
libwebp-base              1.3.2                hcfcfb64_0    conda-forge
libxcb                    1.15                 hcd874cb_0    conda-forge
libxml2                   2.11.5               hc3477c8_1    conda-forge
libzip                    1.10.1               h1d365fa_3    conda-forge
libzlib                   1.2.13               hcfcfb64_5    conda-forge
llvm-meta                 5.0.0                         0    conda-forge
lxml                      4.9.3                    pypi_0    pypi
lz4-c                     1.9.4                hcfcfb64_0    conda-forge
lzo                       2.10              he774522_1000    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
make                      4.3                  h3d2af85_1    conda-forge
metis                     5.1.0             h63175ca_1007    conda-forge
minizip                   4.0.2                h5bed578_0    conda-forge
mkl                       2023.2.0         h6a75c08_50496    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
mumps-seq                 5.2.1               hb3f9cae_11    conda-forge
ninja                     1.11.1               h91493d7_0    conda-forge
numpy                     1.26.0          py311h0b4df5a_0    conda-forge
octomap                   1.9.8                h91493d7_0    conda-forge
ogre                      1.10.12              hbcc8020_2    conda-forge
openal-soft               1.22.2               h2d74725_0    conda-forge
openblas                  0.3.24          pthreads_h3721920_0    conda-forge
opencv                    4.8.1           py311h25e42d1_4    conda-forge
openexr                   3.2.1                h5fba010_0    conda-forge
openh264                  2.3.1                h63175ca_2    conda-forge
openjpeg                  2.5.0                h3d672ee_3    conda-forge
openmp                    5.0.0                    vc14_1    conda-forge
openssl                   3.1.4                hcfcfb64_0    conda-forge
osqp-eigen                0.8.1                h6d7489e_0    conda-forge
pango                     1.50.14              h07c897b_2    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
pip                       23.3.1             pyhd8ed1ab_0    conda-forge
pixman                    0.42.2               h63175ca_0    conda-forge
pkg-config                0.29.2            h2bf4dc2_1008    conda-forge
poppler                   23.10.0              hc2f3c52_0    conda-forge
poppler-data              0.4.12               hd8ed1ab_0    conda-forge
portaudio                 19.6.0               h63175ca_9    conda-forge
postgresql                16.0                 hc80876b_1    conda-forge
proj                      9.3.0                he13c7e8_2    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
pugixml                   1.14                 h63175ca_0    conda-forge
py-opencv                 4.8.1           py311hd5bf162_4    conda-forge
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pyparsing                 3.1.1              pyhd8ed1ab_0    conda-forge
python                    3.11.6          h2628c8c_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.11                    4_cp311    conda-forge
pyyaml                    5.4.1           py311ha68e1ae_4    conda-forge
qpoases                   3.2.1                h0e60522_0    conda-forge
qt-main                   5.15.8              h9e85ed6_17    conda-forge
qwt                       6.2.0                h07be427_6    conda-forge
re2                       2023.06.02           hcbb65ff_0    conda-forge
robot-testing-framework   2.0.1                h63175ca_1    conda-forge
ruby                      3.2.2                h20ad4f3_0    conda-forge
sdl                       1.2.68               h21dd15a_0    conda-forge
sdl2                      2.28.4               h63175ca_0    conda-forge
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
simbody                   3.7                  hade3207_3    conda-forge
simmechanics-to-urdf      0.4.1                    pypi_0    pypi
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.10               hfb803bf_0    conda-forge
soxr                      0.1.3                hcfcfb64_3    conda-forge
spdlog                    1.12.0               h64d2f7d_2    conda-forge
sqlite                    3.44.0               hcfcfb64_0    conda-forge
svt-av1                   1.7.0                h63175ca_0    conda-forge
tbb                       2021.10.0            h91493d7_2    conda-forge
tbb-devel                 2021.10.0            h91493d7_2    conda-forge
tiledb                    2.16.3               hbf04793_3    conda-forge
tiny-process-library      2.0.4                h63175ca_1    conda-forge
tinyxml                   2.6.2                h2d74725_2    conda-forge
tinyxml2                  9.0.0                h0e60522_2    conda-forge
tk                        8.6.13               h5226925_1    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
urdfdom                   3.1.1                h33150cf_0    conda-forge
urdfdom-py                0.3.3                    pypi_0    pypi
urdfdom_headers           1.1.0                h5362a0b_0    conda-forge
uriparser                 0.9.7                h1537add_1    conda-forge
vc                        14.3                h64f974e_17    conda-forge
vc14_runtime              14.36.32532         hdcecf7f_17    conda-forge
vs2015_runtime            14.36.32532         h05e6639_17    conda-forge
vs2019_win-64             19.29.30139         he1865b1_17    conda-forge
vswhere                   3.1.4                h57928b3_0    conda-forge
wb-toolbox                5.6.0              h63175ca_101    robotology
wheel                     0.41.3             pyhd8ed1ab_0    conda-forge
whole-body-controllers    2.5.6              h63175ca_101    robotology
whole-body-estimators     0.10.0             h63175ca_101    robotology
x264                      1!164.3095           h8ffe710_2    conda-forge
x265                      3.5                  h2d74725_3    conda-forge
xerces-c                  3.2.4                h63175ca_3    conda-forge
xorg-kbproto              1.0.7             hcd874cb_1002    conda-forge
xorg-libice               1.1.1                hcd874cb_0    conda-forge
xorg-libsm                1.2.4                hcd874cb_0    conda-forge
xorg-libx11               1.8.7                hefa74cf_0    conda-forge
xorg-libxau               1.0.11               hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xorg-libxext              1.3.4                hcd874cb_2    conda-forge
xorg-libxpm               3.5.17               hcd874cb_0    conda-forge
xorg-libxt                1.3.0                hcd874cb_1    conda-forge
xorg-xextproto            7.3.0             hcd874cb_1003    conda-forge
xorg-xproto               7.0.31            hcd874cb_1007    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
yaml                      0.2.5                h8ffe710_2    conda-forge
yarp-cxx                  3.8.1                hcb6c3dd_7    conda-forge
ycm-cmake-modules         0.15.3               h63175ca_0    conda-forge
zeromq                    4.3.5                h63175ca_0    conda-forge
zlib                      1.2.13               hcfcfb64_5    conda-forge
zstd                      1.5.5                h12be248_0    conda-forge
```

Output of `mamba info`

```
                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.1.0) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


     active environment : robsub
    active env location : C:\Users\apaolino\AppData\Local\mambaforge\envs\robsub
            shell level : 1
       user config file : C:\Users\apaolino\.condarc
 populated config files : C:\Users\apaolino\AppData\Local\mambaforge\.condarc
          conda version : 22.11.1
    conda-build version : not installed
         python version : 3.10.9.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=12.2=0
                          __win=0=0
       base environment : C:\Users\apaolino\AppData\Local\mambaforge  (writable)
      conda av data dir : C:\Users\apaolino\AppData\Local\mambaforge\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\apaolino\AppData\Local\mambaforge\pkgs
                          C:\Users\apaolino\.conda\pkgs
                          C:\Users\apaolino\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\apaolino\AppData\Local\mambaforge\envs
                          C:\Users\apaolino\.conda\envs
                          C:\Users\apaolino\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/22.11.1 requests/2.28.2 CPython/3.10.9 Windows/10 Windows/10.0.22621
          administrator : False
             netrc file : None
           offline mode : False
```

@FabioBergonti @traversaro

--- a/environment.yml
+++ b/environment.yml
@@ -22,7 +22,7 @@ dependencies:
   - pyyaml=5
   - catkin_pkg
   - pip
-  - pugixml 1.14
+  - pugixml=1.14
   - pip:
     - git+https://github.com/ros/urdf_parser_py@31474b9baaf7c3845b40e5a9aa87d5900a2282c3
     - git+https://github.com/robotology/simmechanics-to-urdf@v0.4.1

--- a/environment.yml
+++ b/environment.yml
@@ -22,7 +22,6 @@ dependencies:
   - pyyaml=5
   - catkin_pkg
   - pip
-  - pugixml=1.14
   - pip:
     - git+https://github.com/ros/urdf_parser_py@31474b9baaf7c3845b40e5a9aa87d5900a2282c3
     - git+https://github.com/robotology/simmechanics-to-urdf@v0.4.1

--- a/environment.yml
+++ b/environment.yml
@@ -22,10 +22,7 @@ dependencies:
   - pyyaml=5
   - catkin_pkg
   - pip
-  # Workaround for:
-  # * https://github.com/ami-iit/ironcub-mk1-software/issues/11#issuecomment-1802102145
-  # * https://github.com/conda-forge/openvino-feedstock/issues/52
-  - pugixml=1.13
+  - pugixml 1.14
   - pip:
     - git+https://github.com/ros/urdf_parser_py@31474b9baaf7c3845b40e5a9aa87d5900a2282c3
     - git+https://github.com/robotology/simmechanics-to-urdf@v0.4.1

